src/App.js

```diff
@@ -9,7 +10,14 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <div>
+    <h1>React Article</h1>
+    <Article
+      title={article.title}
+      text={article.text}
+      date={article.date}
```

Comment:
```suggestion
      {...article}
```
It's correct, but here is another way to do it

https://medium.com/@kevinyckim33/jsx-spread-operator-component-props-meaning-3c9bcadd2493